test/framework/src/main/java/org/elasticsearch/index/engine/EngineTestCase.java

```diff
@@ -1137,22 +1137,18 @@ public static void assertOpsOnReplica(
             );
             if (op instanceof Engine.Index) {
                 Engine.IndexResult result = replicaEngine.index((Engine.Index) op);
-                // replicas don't really care to about creation status of documents
-                // this allows to ignore the case where a document was found in the live version maps in
-                // a delete state and return false for the created flag in favor of code simplicity
-                // as deleted or not. This check is just signal regression so a decision can be made if it's
-                // intentional
+                // Replicas don't really care to about creation status of documents. This allows to ignore the case where a document was
+                // found in the live version maps in a delete state and return false for the created flag in favor of code simplicity as
+                // deleted or not. This check is just signal regression so a decision can be made if it's intentional.
                 assertThat(result.isCreated(), equalTo(firstOp));
                 assertThat(result.getVersion(), equalTo(op.version()));
                 assertThat(result.getResultType(), equalTo(Engine.Result.Type.SUCCESS));
 
             } else {
                 Engine.DeleteResult result = replicaEngine.delete((Engine.Delete) op);
-                // Replicas don't really care to about found status of documents
-                // this allows to ignore the case where a document was found in the live version maps in
-                // a delete state and return true for the found flag in favor of code simplicity
-                // his check is just signal regression so a decision can be made if it's
-                // intentional
+                // Replicas don't really care to about found status of documents. This allows to ignore the case where a document was found
+                // in the live version maps in a delete state and return true for the found flag in favor of code simplicity. This check is
+                // just signal regression so a decision can be made if it's intentional.
```

Comment:
```suggestion
                // Replicas don't really care about the "found" status of documents. This allows us to ignore the case where a document was found
                // in the live version maps in a delete state and return true for the found flag in favor of code simplicity. This check is
                // just a simple regression check so a decision can be made if it's intentional.
```

(same caveat as previous comment)